crates/wasm-encoder/src/code.rs

```diff
@@ -59,6 +57,13 @@ impl CodeSection {
         self.num_added += 1;
         self
     }
+
+    /// Add raw bytes as a function body
+    pub fn raw(&mut self, data: &[u8]) -> &mut Self {
+        self.bytes.extend(data);
```

Comment:
This *might* need to encode the length of `data` before just inserting it into `self.bytes`. It depends on if the `data` is already length-prefixed or not. Either way, we should document whether this method expects the data to be prefixed with its length. I think it will be easier in general if this method does the length prefixing itself, but ultimately the higher priority is that we should make sure this method expects whatever `wasmparser` gives us, so that things line up nicely and we don't have to do any intermediate data shuffling.

Reply:
I understand your point. I added the documentation for the method and also for the `CodeSection` example.